src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -187,7 +203,15 @@ export function TravellerSelection({
             </View>
 
             {/* remove new label when requested */}
-            {isOnBehalfOfEnabled && <LabelInfo label="new" />}
+            {isOnBehalfOfEnabled && (
+              <View
+                ref={onBehalfOfIndicatorRef}
+                renderToHardwareTextureAndroid={true}
```

Comment:
I am just curious about this, we haven't used it before.

Reply:
Sometimes the popover doesn't show in Android, we need to add that line based on this source: https://www.npmjs.com/package/react-native-popover-view#show-from-a-ref-not-working

Reply:
Interesting, thanks for the explanation.